### PR TITLE
Table

### PR DESCRIPTION
**Pull request by faraidoun**
i change files

### CODE REVIEW

File: components/icomoon/style.css [hunk 1]
```diff
@@ -1,10 +1,18 @@
 @font-face {
   font-family: 'icomoon';
+<<<<<<< HEAD
```
Comment (Mekaeil) [inline components/icomoon/style.css]: Please Resolve this conflict!

File: components/icomoon/style.css [hunk 2]
```diff
@@ -1,10 +1,18 @@
...
+    url('fonts/icomoon.ttf?uyw8qt') format('truetype'),
+    url('fonts/icomoon.woff?uyw8qt') format('woff'),
+    url('fonts/icomoon.svg?uyw8qt#icomoon') format('svg');
+=======
   src:  url('fonts/icomoon.eot?iv8c3i');
   src:  url('fonts/icomoon.eot?iv8c3i#iefix') format('embedded-opentype'),
     url('fonts/icomoon.ttf?iv8c3i') format('truetype'),
     url('fonts/icomoon.woff?iv8c3i') format('woff'),
     url('fonts/icomoon.svg?iv8c3i#icomoon') format('svg');
+>>>>>>> 56bfbd9f65109267cb6d830ce2a75871e5ed92fc
```
Comment (Mekaeil) [inline components/icomoon/style.css]: Resolve this conflict

File: components/icomoon/style.css [hunk 3]
```diff
@@ -25,9 +33,27 @@
   -moz-osx-font-smoothing: grayscale;
 }
 
+<<<<<<< HEAD
```
Comment (Mekaeil) [inline components/icomoon/style.css]: resolve this conflict

File: sass/_variables.scss [hunk 4]
```diff
@@ -145,22 +145,54 @@
...
+/*========================================================================================*\
+*----------------------------             COLUMN BOX            ----------------------------        
+\*========================================================================================*/
+
+    /*--------------------------------
+    *** CUSTOM COLOR
+    **
+    */
+    $one-handred-persent:100%;
+    $cover-size:cover;
```
Comment (Mekaeil) [inline sass/_variables.scss]: we don't need this variable! the cover is a feature in CSS why you add it to variabl?!

File: sass/_variables.scss [hunk 5]
```diff
@@ -145,22 +145,54 @@
...
+    $height-item:18em;
+    $clear-fix:clear;
```
Comment (Mekaeil) [inline sass/_variables.scss]: here you set variable with "-" and in some line below you set the variables with "_"! why you don't follow the right flow?!
only use one of them, base on the project.

File: sass/modules/item/_column-box.scss [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+    }
+    .triangle-top-right {
+      position: absolute;
+      top: 0em;
+      right: 0em;
+      display: inline-block;
+      width: 0;
+      height: 0;
+      border-style: solid;
+      border-width: 0 200px 200px 0;
```
Comment (Mekaeil) [inline sass/modules/item/_column-box.scss]: please use the "em"

File: sass/modules/table/_simple-responsive-table.scss [hunk 7]
```diff
@@ -13,7 +13,11 @@
     */
 
     .table-continer {
+<<<<<<< HEAD
+      background  : $WHITE;
+=======
       background  : $bgtable;
+>>>>>>> 56bfbd9f65109267cb6d830ce2a75871e5ed92fc
```
Comment (Mekaeil) [inline sass/modules/table/_simple-responsive-table.scss]: Resolve the conflict!

File: sass/modules/table/_simple-responsive-table.scss [hunk 8]
```diff
@@ -35,8 +39,13 @@
 
     .input-area {
 
+<<<<<<< HEAD
+      border        : 2px solid $input-border;
+      border-radius : $in-area-radius;
+=======
```
Comment (Mekaeil) [inline sass/modules/table/_simple-responsive-table.scss]: Resolve the conflict

File: sass/modules/table/_simple-responsive-table.scss [hunk 9]
```diff
@@ -56,7 +65,11 @@
             float     : $left;
             height    : 1.5em;
             margin-top: 0.7em;
+<<<<<<< HEAD
+            opacity   : $img-opacity;
+=======
             opacity   : $opacityimage;
+>>>>>>> 56bfbd9f65109267cb6d830ce2a75871e5ed92fc
```
Comment (Mekaeil) [inline sass/modules/table/_simple-responsive-table.scss]: Resolve the conflict!

File: sass/modules/table/_simple-responsive-table.scss [hunk 10]
```diff
@@ -67,6 +80,11 @@
             width         : 4em;
             margin        : 3px;
             height        : 2.7em;
+<<<<<<< HEAD
+            padding: 0em !important;
+            color: gray;
+=======
+>>>>>>> 56bfbd9f65109267cb6d830ce2a75871e5ed92fc
```
Comment (Mekaeil) [inline sass/modules/table/_simple-responsive-table.scss]: Resolve the conflict!

File: sass/modules/table/_simple-responsive-table.scss [hunk 11]
```diff
@@ -98,12 +116,20 @@
       outline       : inherit;
       color         : white;
       border        : none;
+<<<<<<< HEAD
+      border-radius : 3em;
+=======
       border-radius : 25px;
+>>>>>>> 56bfbd9f65109267cb6d830ce2a75871e5ed92fc
```
Comment (Mekaeil) [inline sass/modules/table/_simple-responsive-table.scss]: Resolve the conflict!

File: sass/modules/table/_simple-responsive-table.scss [hunk 12]
```diff
@@ -98,12 +116,20 @@
...
     }
     
 
     .clear {
+<<<<<<< HEAD
+      clear: both;
+=======
       clear: $bothClear;
+>>>>>>> 56bfbd9f65109267cb6d830ce2a75871e5ed92fc
```
Comment (Mekaeil) [inline sass/modules/table/_simple-responsive-table.scss]: Resolve the conflict

File: sass/modules/table/_simple-responsive-table.scss [hunk 13]
```diff
@@ -123,6 +149,20 @@
...
+      background:#f3f4fb !important;
+    }
+
+    .row:nth-child(odd) {
+      background    : $TR_BG_COLOR;
+    }
+
+    .down-arrows {
+      color       : $BELIZE_HOLE;
+=======
```
Comment (Mekaeil) [inline sass/modules/table/_simple-responsive-table.scss]: Resolve the conflict

File: sass/modules/table/_simple-responsive-table.scss [hunk 14]
```diff
@@ -135,6 +175,7 @@
 
     .down-arrows {
       color       : $bluecolor;
+>>>>>>> 56bfbd9f65109267cb6d830ce2a75871e5ed92fc
```
Comment (Mekaeil) [inline sass/modules/table/_simple-responsive-table.scss]: Resolve the conflict

File: sass/modules/table/_simple-responsive-table.scss [hunk 15]
```diff
@@ -144,6 +185,61 @@
...
+       }
+       &.fulfilled > span::before{
+        background-color: $CIRCEL_GREEN;
+      }
+      &.partially-shipped > span::before{
+        background-color: $CIRCEL_ORANGE;
+      }
+
+     }
+=======
```
Comment (Mekaeil) [inline sass/modules/table/_simple-responsive-table.scss]: Resolve the conflict

File: sass/modules/table/_simple-responsive-table.scss [hunk 16]
```diff
@@ -224,11 +320,18 @@
       margin-right    : 5px;
       background-color: $ORANGE;
     }
+>>>>>>> 56bfbd9f65109267cb6d830ce2a75871e5ed92fc
 
     .table {
       overflow-x: auto;
     }
 
+<<<<<<< HEAD
```
Comment (Mekaeil) [inline sass/modules/table/_simple-responsive-table.scss]: Resolve the conflict

File: sass/modules/table/_simple-responsive-table.scss [hunk 17]
```diff
@@ -224,11 +320,18 @@
...
+    .table{
+      overflow-y: hidden;
+    }
+=======
+>>>>>>> 56bfbd9f65109267cb6d830ce2a75871e5ed92fc
```
Comment (Mekaeil) [inline sass/modules/table/_simple-responsive-table.scss]: Resolve the conflict

File: sass/modules/table/_simple-responsive-table.scss [hunk 18]
```diff
@@ -276,9 +379,12 @@
       .control-indicator {
         left: 0 !important;
       }//for mobiles
+<<<<<<< HEAD
+=======
       .right-button {
         float: left;
       }
+>>>>>>> 56bfbd9f65109267cb6d830ce2a75871e5ed92fc
```
Comment (Mekaeil) [inline sass/modules/table/_simple-responsive-table.scss]: Resolve the conflict

File: sass/modules/table/_simple-responsive-table.scss [hunk 19]
```diff
@@ -312,10 +418,21 @@
...
       /*--------------------------------
       ***   
       **   HEADER STYLE
       */
+<<<<<<< HEAD
+      
+      .control-indicator{
+        right: 0;
+=======
```
Comment (Mekaeil) [inline sass/modules/table/_simple-responsive-table.scss]: Resolve the conflict

File: sass/modules/table/_simple-responsive-table.scss [hunk 20]
```diff
@@ -409,7 +527,11 @@
     }// rtl
 
 
+<<<<<<< HEAD
+    /*--------------------------------
```
Comment (Mekaeil) [inline sass/modules/table/_simple-responsive-table.scss]: Resolve the conflict

File: sass/style.scss [hunk 21]
```diff
@@ -51,10 +51,19 @@
...
         @import "modules/item/menu-food";
 
+<<<<<<< HEAD
+        @import "modules/item/column-box";
+
+        @import"modules/table/simple-responsive-table";
+
+        @import "modules/item/bg-with-widgets";
+
+=======
```
Comment (Mekaeil) [inline sass/style.scss]: Resolve the conflict

File: sass/style.scss [hunk 22]
```diff
@@ -51,10 +51,19 @@
...
         @import "modules/item/bg-with-widgets";
 
         @import "modules/table/simple-responsive-table";
 
+>>>>>>> 56bfbd9f65109267cb6d830ce2a75871e5ed92fc
```
Comment (Mekaeil) [inline sass/style.scss]: Resolve the conflict

File: sass/widget/_checkbox_radio_buttons.scss [hunk 23]
```diff
@@ -39,7 +39,11 @@
 **
 */
 .control input:checked ~ .control-indicator {
+<<<<<<< HEAD
+  background:$CIRCEL_BLUE;
+=======
```
Comment (Mekaeil) [inline sass/widget/_checkbox_radio_buttons.scss]: Resolve the conflict

File: simple-responsive-table.html [hunk 24]
```diff
@@ -1,6 +1,9 @@
 <!DOCTYPE html>
 <html lang="en">
+<<<<<<< HEAD
```
Comment (Mekaeil) [inline simple-responsive-table.html]: Resolve the conflict

File: simple-responsive-table.html [hunk 25]
```diff
@@ -22,7 +25,11 @@
                 <div class="input-area">
                     <span class="icon-filter img-icon"></span>
                     <input type="text" class="text-box" placeholder="Filter sales orders">
+<<<<<<< HEAD
```
Comment (Mekaeil) [inline simple-responsive-table.html]: Resolve the conflict

File: simple-responsive-table.html [hunk 26]
```diff
@@ -71,9 +78,15 @@
                         
                         <div class="td">Gaspar Antunes</div>
                         
+<<<<<<< HEAD
+                            <div class="td  status fulfilled">
+                                <span>FULFILLED</span>
```
Comment (Mekaeil) [inline simple-responsive-table.html]: Resolve the conflict

File: simple-responsive-table.html [hunk 27]
```diff
@@ -125,8 +144,13 @@
                         
                         <div class="td">Trashae Hubbard</div>
                         
+<<<<<<< HEAD
+                        <div class="td  status partially-shipped">
+                            <span>PARIALLY SHIPPED</span>
+=======
                         <div class="td">
                             <div class="yellow"><span class="doty"></span>PARIALLY SHIPPED</div>
+>>>>>>> 56bfbd9f65109267cb6d830ce2a75871e5ed92fc
```
Comment (Mekaeil) [inline simple-responsive-table.html]: Resolve the conflict

File: simple-responsive-table.html [hunk 28]
```diff
@@ -174,8 +203,13 @@
                        
                         <div class="td">Ngo Hai Giang</div>
                         
+<<<<<<< HEAD
+                        <div class="td  status confirmed">
+                            <span>CONFIRMED</span>
+=======
                         <div class="td">
                             <div class="blue"><span class="dotb"></span>CONFIRMED</div>
+>>>>>>> 56bfbd9f65109267cb6d830ce2a75871e5ed92fc
```
Comment (Mekaeil) [inline simple-responsive-table.html]: Resolve the conflict

File: simple-responsive-table.html [hunk 29]
```diff
@@ -224,8 +263,13 @@
                         
                         <div class="td">Leslee Moss</div>
                         
+<<<<<<< HEAD
+                        <div class="td  status partially-shipped">
```
Comment (Mekaeil) [inline simple-responsive-table.html]: Resolve the conflict

File: simple-responsive-table.html [hunk 30]
```diff
@@ -552,8 +651,13 @@ <h2 class="title">RTL Simple responsive table</h2>
                             
                             <div class="td">بانه ورلد</div>
                             
+<<<<<<< HEAD
+                            <div class="td  status fulfilled">
+                                <span>تکمیل شده</span>
+=======
                             <div class="td">
                                 <div class="green"><span class="dotg"></span>تکمیل شده</div>
+>>>>>>> 56bfbd9f65109267cb6d830ce2a75871e5ed92fc
```
Comment (Mekaeil) [inline simple-responsive-table.html]: Resolve all of the conflicts in page...

**Comment by faraidoun** [comment]
pleas see changes

**Comment by faraidoun** [comment]
pleas see changes. i change varible names

**Pull request merged**